### ISSUE TITLE
Option for locked images/videos to maintain size (fit screen) and mousewheel to continue to control video position

### ISSUE THREAD

**Issue by but-a-thought** [extesy/hoverzoom#1374]
**Is your feature request related to a problem? Please describe.**

Often when I am changing the video position with the mouse wheel, while hovering over a link, if the mouse moves even a little, it reloads the video because HoverZoom+ seem to think it is a new link to load, causing the video to restart (video linked below shows this behavior). This happens even when the mouse cursor maintains position over the link text.

This appears to happen most often with twitch video clips in Firefox at https://old.reddit.com/r/LivestreamFails/
Doesn't appear to happen with v.redd.it videos.

On Chrome, it seems to happen less often but it is inconsistent (happens for some twitch clip links and not for others - see video below).

Locking the video with action key "**Lock image on screen**" helps prevent this, but when you "Lock image on screen" it makes the image/video go to full size, and reverts the mousewheel to controlling the size of the image/video.

**Describe the solution you'd like**
Allow the option to have locked video/image just fit screen (same size they are when you hover), and still allow mousewheel to control video position. Just having it maintain size would fine as well but still allowing mouse wheel to control position would be nice.

**Additional context**
I wasn't sure if to make this a bug or feature request, I suppose it is bit of both.

Below is a video of the behavior in Google Chrome - you can see it working fine with one link, and doing the reloading of video with any movement in the second link.

https://github.com/extesy/hoverzoom/assets/243932/841b1af8-95cf-4da1-9b3d-aea5dd53bb28

**Comment by extesy** [comment]
@GrosPoulet you seem to like tinkering with image display options, perhaps you could take a look at this request?

**Comment by GrosPoulet** [comment]
@extesy i'll give it a shot this week

**Comment by LiliaDoe** [comment]
@extesy I have a PR that changes the "lock image" function so that the first press of it always makes the image fit the screen. A second press of it while the image is locked will then zoom in, opposite of it now. Would that be a good solution to this?

**Comment by extesy** [comment]
Can you please clarify - what's the difference between "making an image fit the screen" and "zooming in"? Do you mean making it larger than the screen size?

**Comment by LiliaDoe** [comment]
Yes, that's what I meant. Currently, some images when locked will appear larger than the window, and pressing the lock button again would make it fit to the window. This would swap it so locking the image would fit it to the screen first, while a second press would make it larger

**Comment by extesy** [comment]
I think that updated behavior makes sense, I would prefer it over the original.
